webapp/packages/core-authentication/src/AuthProvidersResource.ts

```diff
@@ -44,6 +44,7 @@ export class AuthProvidersResource extends CachedMapResource<string, AuthProvide
   ) {
     super();
 
+    this.preloadResource(this.serverConfigResource, () => undefined);
     this.sync(
       serverConfigResource,
```

Comment:
sync already preloads resource

Reply:
so you don't need to add `this.preloadResource(this.serverConfigResource, () => undefined);` or load `serverConfigResource` in loader manually